### DISCUSSION TITLE
Install user_migration App by default

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#1179] [Ideas] [answered]
https://github.com/nextcloud/all-in-one/blob/ef0361dd29543451ad694f24e72a9886de50fca1/Containers/nextcloud/entrypoint.sh#L231
As there are some apps installed by default, in my opinion, it also a good idea to install the user_migration app by default on a new aio instance. I think the reason for this is self explaining.

**Comment by szaimen** [comment, marked as answer]
We are not going to add this, however I am thinking about a way to adjust the apps that get installed on the first startup. See https://github.com/nextcloud/all-in-one/issues/1198
